typography.ftd

```diff
@@ -1,124 +1,174 @@
+-- or-type display-type:
+
+-- constant integer inline: 1
+-- constant integer block: 2
```

Comment:
This variant is not needed. As per your implementation, this does the same thing as `text-center`.